langroid/language_models/openai_gpt.py

```diff
@@ -671,11 +671,12 @@ def set_stream(self, stream: bool) -> bool:
         return tmp
 
     def get_stream(self) -> bool:
-        """Get streaming status"""
+        """Get streaming status. Note we disable streaming in quiet mode."""
         return (
             self.config.stream
             and settings.stream
             and self.config.chat_model not in NON_STREAMING_MODELS
+            and not settings.quiet
```

Comment:
@nilspalumbo I am disabling LLM streaming when we are in quiet mode. Let me know if you see any issues with that.

Reply:
That sounds reasonable to me!